source/macos/AdaptiveCards/AdaptiveCards/Views/ACRTextField.swift

```diff
@@ -31,15 +57,24 @@ class ACRTextField: NSTextField {
     }
     
     private func setupConstraints() {
+        translatesAutoresizingMaskIntoConstraints = false
         heightAnchor.constraint(equalToConstant: config.height).isActive = true
-        if config.wantsClearButton {
+        if wantsClearButton {
             clearButton.trailingAnchor.constraint(equalTo: trailingAnchor, constant: -config.rightPadding).isActive = true
             clearButton.centerYAnchor.constraint(equalTo: centerYAnchor).isActive = true
         }
     }
     
     private (set) lazy var clearButton: NSButtonWithImageSpacing = {
-        let view = NSButtonWithImageSpacing(image: config.clearButtonImage ?? NSImage(), target: self, action: #selector(handleClearAction))
+        let clearImage: NSImage?
+        if let isDarkMode = isDarkMode, config.clearButtonImage == nil && wantsClearButton {
+            // displaying old clear button
```

Comment:
```suggestion
        if let isDarkMode = isDarkMode, config.clearButtonImage == nil, wantsClearButton {
            // displaying old clear button
```
',' and '&&' are both same. :)